contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -37,6 +38,12 @@ contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions
     bytes returnData
   );
 
+  event RequestNotProcessed(
```

Comment:
I'm thinking that adding donID here would help.

Reply:
Don ID might be hard to get, there is no reverse lookup. How about just Coordinator address?

Reply:
That works too. I'll check with @pinebit